Doc/library/shutil.rst

```diff
@@ -445,6 +445,8 @@ Directory and files operations
 
    Return the path to an executable which would be run if the given *cmd* was
    called.  If no *cmd* would be called, return ``None``.
+   If *cmd* contains a directory component, *which* will only check the specified path directly
+   and will not search the directories listed in *path* or in the system's ``PATH`` environment variable.
```

Comment:
```suggestion
   If *cmd* contains a directory component, :func:`!which` only checks the
   specified path directly and does not search the directories listed in *path*
   or in the system's ``PATH`` environment variable.
```

Can we put this one after *path* is being documented? and make it under a `.. note::` directive please? TiA.